independent-projects/resteasy-reactive/client/runtime/src/main/java/org/jboss/resteasy/reactive/client/impl/UniInvoker.java

```diff
@@ -0,0 +1,21 @@
+package org.jboss.resteasy.reactive.client.impl;
+
+import io.smallrye.mutiny.Uni;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.GenericType;
+
+public class UniInvoker extends AbstractRxInvoker<Uni<?>> {
+
+    private WebTargetImpl target;
+
+    public UniInvoker(WebTargetImpl target) {
+        this.target = target;
+    }
+
+    @Override
+    public <R> Uni<R> method(String name, Entity<?> entity, GenericType<R> responseType) {
+        AsyncInvokerImpl invoker = (AsyncInvokerImpl) target.request().rx();
+        return Uni.createFrom().completionStage(invoker.method(name, entity, responseType));
```

Comment:
I wonder if there is an expectation that anything `Uni` should be lazy and not cached @cescoffier?

This sort of set-up makes it eager and cached.